_posts/2018-10-11-migrating-from-galen-to-backstop.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: post
+title:  "Migrating from Galen to Backstop"
+date: 2018-10-22 14:55:10 +0200
+categories: testing
+author_name: Jacek Smolak
+author_url : /author/jaceksmolak
+author_avatar: jaceksmolak
+read_time : 12
+show_related_posts: false
+square_related:
+feature_image: posts/2018-10-25-migrating-from-galen-to-backstop/migration.jpg
+---
+
+> Visual regression testing has recently started gaining popularity- and for good reason. At its most basic, it's a series of tests that run through your site, take screenshots of various components, compare those screenshots against a baseline, and alert you when things change.
+
+_(Css-tricks; [Visual Regression Testing with PhantomCSS](https://css-tricks.com/visual-regression-testing-with-phantomcss/); November 17,2015)_
+
+## The reason behind the change
+
+[HolidayCheck](https://www.holidaycheck.de/), as any other company, needs to run its business not worrying that some changes might break its product. One of the ways, of maintaining that peace of mind, is to ensure that visually everything is A-OK.
+
+How did we do that? Our project used [Galen](http://galenframework.com/) as a test runner. Tests were executed on virtual machines, provided by external partner. At the beginning everything worked smoothly (OK, perhaps not, but it was maintained by few nice engineers, so that the rest of us could use that tool). Yet, couple of months ago, things started to break badly. For the reasons, not clear to us, [Selenium](https://www.seleniumhq.org/) started failing, randomly, more and more often. Also, tests were being executed slower and slower. Up to the point where the whole CI pipeline took ~30min to finish. Yes, thirty.
```

Comment:
`nice engineers` - I would replace with `good engineers`

Reply:
The whole paragraph will be rewritten very soon. It will be removed. Less detailed info will be there.